package/MDAnalysis/coordinates/__init__.py

```diff
@@ -567,8 +562,20 @@
 
 **Optional attributes**
 
+ ``delta``
+     integrator time step (in native units); hence the "length"
+     of a trajctory frame is  ``skip_timestep*delta`` time units
  ``compressed``
      string that identifies the compression (e.g. "gz" or "bz2") or ``None``.
+ ``fixed``
+     bool, saying if there are fixed atoms (e.g. dcds)
+ ``periodic``
```

Comment:
Boolean, if ``True`` then it is know that the simulation was performed with periodic boundary conditions (and the ``dimensions`` attribute should contain a valid simulation cell.